ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1551,6 +1558,7 @@ void ConfigFdbTableEntry(struct mac_learning_info learn_info,
     if (!status.ok())
       printf("%s: Failed to program l2_fwd_rx_table\n",
```

Comment:
This should be removed or replaced by a logger call.

Reply:
right now, ovs-sidecar doesnt support any logging mechanism. logging was never implemented.